common/src/main/java/org/apache/uniffle/common/netty/TransportFrameDecoder.java

```diff
@@ -67,9 +67,17 @@ public void channelRead(ChannelHandlerContext ctx, Object data) {
       if (frame == null) {
         break;
       }
-      Message msg = Message.decode(curType, frame);
-      if (msg.body() == null) {
-        frame.release();
+      Message msg = null;
+      try {
+        msg = Message.decode(curType, frame);
+      } finally {
+        boolean shouldRelease =
```

Comment:
Should we release the frame if the msg is `null`?

Reply:
Yes, we should. Although this can barely happen, we still need to fix this. Done.